test/utils/toSpecPath.test.ts

```diff
@@ -0,0 +1,19 @@
+import * as assert from 'assert';
+
+import toSpecPath from '../../src/utils/toSpecPath';
+
+suite("toSpecPath", () => {
+    test("Converting shallow path", () => {
+        assert.equal(
+            'spec/controllers/test_controller_spec.rb',
+            toSpecPath('app/controllers/test_controller.rb')
+        );
+    });
+
```

Comment:
it would be nice to have a test with entry argument `spec/controllers/test_controller_spec.rb`, in case the opened file is already `spec`.